cli-feed-v3.json

```diff
@@ -124,6 +124,16 @@
       "sha2": "DC1D0F2D2B5817E27B78CF5109EA086F865D76867ADD3FD510287CA92754FFE8",
       "FUNCTIONS_EXTENSION_VERSION": "beta"
     },
+    "2.2.2": {
+      "Microsoft.NET.Sdk.Functions": "1.0.14",
+      "cli": "https://functionscdn.azureedge.net/public/2.0.1-beta.30/Azure.Functions.Cli.win-x86.2.0.1-beta.31.zip",

```

Comment:
The path is incorrect
*public/2.0.1-beta.31/*

Reply:
thanks for catching that